src/kudu/tserver/tserver_admin.proto

```diff
@@ -27,14 +27,18 @@ enum TSTabletManagerStatePB {
   // Indicates that Tablet Manager is initializing.
   MANAGER_INITIALIZING = 0;
 
-  // Indicates that Tablet Manager is running and can create new
-  // tablets.
-  MANAGER_RUNNING = 1;
+  // Indicates that Tablet Manager is initialized and
+  // Raft data structures e.g. consensus and log have been created.
+  // However Raft has not yet started
+  MANAGER_INITIALIZED = 1;
```

Comment:
Initially the manager is constructed but not usable. INITIALIZED is the
point where we have built a set of datastructures and RUNNING is when raft is running.
Having these states help us put nice asserts and throw out RPC calls if the state is lower than what it can handle.

Reply:
I don't think we need too fine a distinction, i.e. INITIALIZING and INITIALIZED can be merged together.